README.md

```diff
@@ -145,21 +145,30 @@ influxdb.write_point(name, data)
 influxdb.write_point(name, data, time_precision)
 ```
 
-> ### A Note About Time Precision
->
-> The default precision (in this Rubygem) is `"s"` (second), as Ruby's
-> `Time#to_i` operates on this resolution.
->
-> If you write data points with sub-second resolution (for example
-> via `Time#to_r`), you need to configure your client instance with an
-> appropiate `time_precision` option **and** you'll need to provide a
-> timestamp value which reflects this precision:
->
-> ```ruby
-> influxdb = InfluxDB::Client.new time_precision: "ms"
-> time = (Time.now.to_r * 1000).to_i
-> influxdb.write_point("foobar", { values: { n: 42 }, timestamp: time })
-> ```
+### A Note About Time Precision
+
+The default precision in this gem is `"s"` (second), as Ruby's
+`Time#to_i` operates on this resolution.
+
+If you write data points with sub-second resolution,
+you _have_ to configure your client instance with
+a more granular `time_precision` option **and** you need to provide
+timestamp values which reflect this precision.
+
+**If you don't do this, your points will be lost!**
```

Comment:
Scratch the paragraph seperation, otherwise the sentence might loose its context.

It would be nice to have further details, like this FAQ entry explaning [how InfluxDB handles duplicate points](https://docs.influxdata.com/influxdb/v1.3/troubleshooting/frequently-asked-questions/#how-does-influxdb-handle-duplicate-points). I'd go so far and copy the first paragraph from there (emphasis mine):

> A point is uniquely identified by the measurement name, tag set, **and timestamp**.
> If you submit a new point with the same measurement, tag set, and timestamp as
> an existing point, the field set becomes the union of the old field set and the new
> field set, where any ties go to the new field set. **This is the intended behavior.**